filebeat/module/iis/_meta/docs.asciidoc

```diff
@@ -0,0 +1,5 @@
+== iis module
+
+This is the iis module for Filebeat for use with Internet Information Services (IIS) for Windows OS.
```

Comment:
This documentation refers to "the iis module". This should read "the IIS module" (note the capitalisation of "IIS"). This documentation could also be a bit more descriptive. See, for example, [my own attempting at this file](https://github.com/cgwrench/beats/blob/iis-filebeat-module/filebeat/module/iis/_meta/docs.asciidoc).